database/migrations/2021_05_28_114212_create_collections_table.php

```diff
@@ -17,22 +17,7 @@ public function up()
             $table->id();
             $table->string('handle');
             $table->string('title');
-            $table->json('routes')->nullable();
-            $table->boolean('dated')->default(false);
-            $table->string('past_date_behavior')->nullable();
-            $table->string('future_date_behavior')->nullable();
-            $table->boolean('default_publish_state')->default(true);
-            $table->boolean('ampable')->default(false);
-            $table->json('sites')->nullable();
-            $table->string('template')->nullable();
-            $table->string('layout')->nullable();
-            $table->string('sort_dir')->nullable();
-            $table->string('sort_field')->nullable();
-            $table->string('mount')->nullable();
-            $table->json('taxonomies')->nullable();
-            $table->boolean('revisions')->default(false);
-            $table->json('inject')->nullable();
-            $table->json('structure')->nullable();
+            $table->json('settings')->nullable();
```

Comment:
But this will mean anyone upgrading won’t get the new code. I would revert this change.

Reply:
I can definitely see where you're coming from, but as this is a "clean" PR with new functionality, I'd rather opt for existing users applying a composer patch we can provide in docs instead, as they will be the exception.

Reply:
They won’t. So many people use v0.2 of this. There is no benefit in squashing the migration that I can see. I didn’t do it originally intentionally.

Reply:
But v0.2 is only entries? Your PR provides migrations for everything else, so they'd have to use your branch from before February 15th 2022 for the change to be of any effect.

Reply:
Yeah valid point, in my head v2 also handled collections but I double checked and youre correct.